docs/customization/customizing-your-storefront.mdx

```diff
@@ -0,0 +1,170 @@
+that partial:
+
+```html
+<div class="hero__container">
+  <%= image_tag('hero-image') %>
+  <div class="hero__message">
+    ...
+  </div>
+</div>
+```
+
+If we test it out, we'll find that we need to add the hero image to the asset pipeline:
+
+![Missing hero image error](/img/customizing-your-storefront/missing-hero-image-error.png)
+
+## Running the frontend specs
+
+If you try and run the [RSpec][] test suite at this point, you may also see some specs fail because
+of this error:
+
+```
+$ bundle exec rspec spec/requests/spree/home_spec.rb
+
+...
+
+Failures:
+
+  1) Home layout provides current user to the searcher class
+     Failure/Error: <%= image_tag('hero-image') %>
+
+     ActionView::Template::Error:
+       The asset "hero-image" is not present in the asset pipeline.
+     # ./app/views/spree/home/_hero.html.erb:3:in `_app_views_spree_home__hero_html_erb___8794442347239906_218740'
+     # ./app/views/spree/home/index.html.erb:1:in `_app_views_spree_home_index_html_erb__1597336803422850061_218720'
+     # ./spec/requests/spree/home_spec.rb:18:in `block (2 levels) in <top (required)>'
+     # ------------------
+     # --- Caused by: ---
+     # Sprockets::Rails::Helper::AssetNotFound:
+     #   The asset "hero-image" is not present in the asset pipeline.
+     #   ./app/views/spree/home/_hero.html.erb:3:in `_app_views_spree_home__hero_html_erb___8794442347239906_218740'
+
+...
+
+```
+
+When you installed SolidusStarterFrontend to your app, you also received some some specs for testing
+the frontend. These specs are part of the SolidusStarterFrontend test suite itself, that is, they
+are used in testing the SolidusStarterFrontend project. By copying these specs to your app, you get
+a good starting test suite for testing your application.
+
+## Adding an image
+
+We'll be using this image as the hero image for the home page:
+
+![Hero image](/img/customizing-your-storefront/hero-image.png)
+
+Following the [Rails Assets Pipeline guide][], we're supposed to save this image in the
+`app/assets/images` directory. Once we save the image there, the home page now looks like this:
+
+![Home page with unstyled hero](/img/customizing-your-storefront/home-page-with-unstyled-hero.png)
+
+As you can see, we'll need to update the stylesheets in order to move the image above the hero
+message.
+
+## Customizing a stylesheet
+
+If we search for the word "hero" in the stylesheets of the app, we would find
+`app/assets/stylesheets/spree/frontend/components/presentation/_hero.scss`:
+
+```scss
+.hero {
+  background: $color-background-dark;
+  color: $color-copy-bright;
+  padding: 10rem 0;
+
+  &__container {
+    @extend %section-spacings;
+    display: flex;
+  }
+}
+```
+
+Like the hero partial from earlier, this stylesheet was copied to the Rails app when SolidusStarterFrontend was installed on the app.
+
+SolidusStarterFrontend stylesheets are written with [Sass][].
```

Comment:
This looks a bit off here. Can we try to add this information in the previous paragraph? Something like:

> Like the hero partial from earlier, this stylesheet (written with [Sass][]) was copied to the Rails app when SolidusStarterFrontend was installed on the app.

Reply:
I changed it to "Like the hero partial from earlier, this [Sass][] sstylesheet was copied to the Rails app when SolidusStarterFrontend was installed on the app."